charts/ocis/templates/policies/deployment.yaml

```diff
@@ -73,11 +73,6 @@ spec:
               value: "" # no cert needed
               {{- end }}
             {{- end }}
-            - name: POLICIES_JWT_SECRET
```

Comment:
Was this really removed in alpha 5?

Reply:
no, technically not removed. but unreferenced = not used. Actually it was NEVER used. https://github.com/owncloud/ocis/pull/7893 removes the leftovers

Reply:
Ok